nix/README.md

```diff
@@ -61,6 +61,20 @@ nix-shell --run postgrest-test-spec-postgresql-9.5.21
 
 ```
 
+## Linting and styling code
+
+The nix-shell also contains scripts for linting and styling the PostgREST
+source code:
+
+```bash
+# Linting
+nix-shell --run postgrest-lint
```

Comment:
I didn't know that when inside the `nix-shell` I could just run `postgrest-lint`.

I now understand what you meant about autocompleting with tab when doing `postgrest-test-spec-<tab>`. This is really nice  :)

Reply:
(I thought autocomplete was meant to work like `nix-shell --run postgrest-test-spec-<tab>`)